lib/Models/exercise.dart

```diff
@@ -3,18 +3,26 @@ import 'package:exerlog/Models/sets.dart';
 import 'package:exerlog/UI/exercise/totals_widget.dart';
 
 class Exercise {
+
+  Exercise(this.name, this.sets, this.bodyParts, this.totalSets, this.totalReps,
+      this.totalWeight);
+
+  Exercise.fromJson(Map<String, Object?> exercise)
```

Comment:
This seems like a weird way of doing this to me. Maybe make this a factory constructor instead.

Reply:
I see you just moved this coded, so maybe we should keep this in mind for another PR

Reply:
I tried not to change code in PR like this, maybe another time :)